src/confdb/data/SmartPrescaleTable.java

```diff
@@ -204,7 +204,6 @@ public void update() {
             StringTokenizer pathTokens = new StringTokenizer(strCondition, "/ ");
             while (pathTokens.hasMoreTokens()) {
                 String strPath = pathTokens.nextToken().trim();
-                if (strPath.length() < 5) continue;
```

Comment:
I'm not sure I understand what this was doing. It looks like it effectively caught strings like `NOT / AND / OR / ( / ) / TRUE` up to now (but obviously, this check was not enough for "MASKING"; maybe I should have used "BUT"..).